src/main/java/io/supertokens/webserver/api/passwordless/CheckCodeAPI.java

```diff
@@ -88,6 +90,22 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws I
             JsonObject result = new JsonObject();
             result.addProperty("status", "OK");
 
+            if (getVersionFromRequest(req).greaterThanOrEqualTo(SemVer.v5_0)) {
```

Comment:
if check not required since this is a new API anyway

Reply:
why is this extra info required?

Reply:
decided to add